setup.cfg

```diff
@@ -5,12 +5,12 @@ maintainer_email = admin@tskit.dev
 description = Quantitative trait simulation of tree sequence data
 long_description = file: README.md
 long_description_content_type = text/markdown
-url = https://tskit.dev/
+url = https://tskit.dev/docs/
```

Comment:
```suggestion
url = https://tskit.dev/tstrait/
```

Reply:
I just made the changes, and thank you for your suggestions.